test/recipes/80-test_cmp_http.t

```diff
@@ -265,28 +269,32 @@ sub load_tests {
     return \@result;
 }
 
-sub mock_server_pid {
-    return `lsof -iTCP:$server_port` =~ m/\n\S+\s+(\d+)\s+[^\n]+LISTEN/s ? $1 : 0;
-}
-
 sub start_mock_server {
     my $args = $_[0]; # optional further CLI arguments
     my $dir = bldtop_dir("");
-    my $cmd = "LD_LIBRARY_PATH=$dir DYLD_LIBRARY_PATH=$dir " .
-        bldtop_dir($app) . " -config server.cnf $args";
-    my $pid = mock_server_pid();
-    if ($pid) {
-        print "Mock server already running with pid=$pid\n";
-        return $pid;
-    }
+    local $ENV{LD_LIBRARY_PATH} = $dir;
+    local $ENV{DYLD_LIBRARY_PATH} = $dir;
+    my $cmd = bldtop_dir($app) . " -port $server_port -config server.cnf $args";
```

Comment:
The `-port $server_port` part is not needed here 
and actually confusing because `$server_port` is not yet known at this point.

Reply:
So you mean that I should make that `-port 0`?

Reply:
You can simply leave out the new `-port $server_port` part because `server.cnf` anyway contains `port = 0`, which provides the right default value `0`, and there is no need to override that at CLI level using `-port`.

Reply:
Don't forget to remove the misleading `-port $server_port`.

Reply:
Done